packages/styles/src/modules/pagination.css

```diff
@@ -0,0 +1,191 @@
+/**
+ * Used to split large content into several pages, allowing users to navigate between them instead of displaying all information on a single page.
+ * @name sd-pagination
+ * @status stable
+ * @since 1.0
+ * @variant { simple } sd-pagination--... The pagination format.
+ * @boolean sd-pagination--inverted Inverts the pagination style.
+ */
+
+.sd-pagination {
+  > :not(ul) {
+    position: absolute;
+    overflow: hidden;
+    height: 1px;
+    width: 1px;
+  }
+
+  ul {
+    @apply flex items-center gap-2;
+
+    li a {
+      @apply text-primary flex items-center justify-center;
```

Comment:
There should be some hover styles I believe. Please checkout `sd-interactive` which should be followed.